docs/DeveloperGuide.md

```diff
@@ -69,7 +69,41 @@ Eslint and its accompaning modules can be installed through NPM, so do ensure th
 ### Building and running RepoSense from code
 
 1. Execute the following command on the OS terminal inside the project directory. <br/>
-Usage: `gradlew run -Dargs="([-config CONFIG_FOLDER] | [-repos REPO_PATH_OR_URL...]) [-view [REPORT_FOLDER]] [-output OUTPUT_DIRECTORY] [-since DD/MM/YYYY] [-until DD/MM/YYYY] [-formats FORMAT...] [-isac | --ignore-standalone-config]"` <br/>
+Usage: `gradlew run -Dargs="([--config CONFIG_FOLDER] | [--repos REPO_PATH_OR_URL...]) [--view [REPORT_FOLDER]] [--output OUTPUT_DIRECTORY] [--since DD/MM/YYYY] [--until DD/MM/YYYY] [--formats FORMAT...] [--ignore-standalone-config]"` <br/>
+Named Arguments: 
+```
+--help, -h           Show help message.
+--view [PATH], -v [PATH]
+                     Starts a server to  display  the  dashboard in the
+                     provided directory.If  used  as  a  flag  (with no
+                     argument), generates  a  report  and automatically
```

Comment:
Extra whitespaces (more than 1) between words throughout each description (missing a whitespace in `directory.If` here though